code/game/machinery/doors/poddoor.dm

```diff
@@ -14,12 +14,11 @@
 	else
 		return FALSE
 
-/obj/machinery/door/poddoor/attackby(obj/item/I, mob/user, params)
-	. = ..()
-
-	if(!I.pry_capable)
-		return
+/obj/machinery/door/poddoor/attackby(obj/item/W, mob/user)
+	return //BLAST DOORS OVERRIDE
 
+/obj/machinery/door/poddoor/crowbar_act(mob/living/user, obj/item/W)
+	add_fingerprint(user)
```

Comment:
This will runtime. Either make it handled trough proper parent logic (preferred) or make it work properly.